openMINDS_pipeline/vocab.py

```diff
@@ -78,7 +80,12 @@ def __init__(self, directory_structure: DirectoryStructure):
     def _load_types(self):
         if os.path.exists(self._types_file):
             with open(self._types_file, "r") as types_f:
-                return json.load(types_f)
+                jsond = json.load(types_f)
+                cleaned_jsond = {}
```

Comment:
`jsond` would be better as `_types`, since we care about what is in the data structure, not its form.
`cleaned_...` is not an appropriate name, since we are just removing the namespace. Perhaps `_type_names`?